### PR TITLE
asset sandbox

### PR DESCRIPTION
**Pull request by briwylde08**

### CODE REVIEW

File: docs/tools-and-sdks.mdx [hunk 1]
```diff
@@ -19,6 +19,10 @@ Allows other Stellar apps to request transaction signing or identity verificatio
 
 View all anchors on Stellar, their currencies, and where they operate.
 
+### [Asset Sandbox](https://stellar.cheesecakelabs.com/?utm_source=stellar&utm_medium=landing-page&utm_campaign=stellar-asset-sandbox)
```
Comment (ElliotFriend) [inline docs/tools-and-sdks.mdx:22]: I be the answer is "yes," but feel like I should ask: Do we want the `?utm_source=stell....` included in the link?

File: docs/tools-and-sdks.mdx [hunk 2]
```diff
@@ -19,6 +19,10 @@ Allows other Stellar apps to request transaction signing or identity verificatio
...
+
+A sandbox supported by SDF and Cheesecake Labs for businesses to experiment with asset issuance on the Stellar's test network.
```
Comment (ElliotFriend) [inline docs/tools-and-sdks.mdx:24]:
```suggestion
A sandbox supported by SDF and Cheesecake Labs for businesses to experiment with asset issuance on Stellar's test network.
```

nitpick: `the` or `'s` feel... "conflicting" ?? not sure how to explain it but it reads to me like there should be one or the other, but not both

Also, would we want to link to Cheesecake Labs, too? i.e., `... SDF and [Chesecake Labs](https://cheesecakelabs.com/) for ...`

**Review by ElliotFriend — COMMENTED** [review]
Probly **way** more feedback than is actually needed for a simple addition like this lol

**Review by ElliotFriend — APPROVED** [review]

File: docs/tools-and-sdks.mdx (hunk 1 above)
Comment (briwylde08) [reply docs/tools-and-sdks.mdx:22]: I'm not sure? I just copied the link from the blog https://stellar.org/blog/app-for-issuing-tokens-assets-on-stellar-testnet

**Pull request merged**